docs/upgrade_tails_usbs.rst

```diff
@@ -0,0 +1,124 @@
+Updating Tails USBs
+===================
+
+There are 3 components of SecureDrop that use Tails USBs:
+
+  #. The *Admin Workstation* (allowed to connect to the Internet)
+  #. The *Journalist Workstation* (allowed to connect to the Internet)
+  #. The *Secure Viewing Station* (airgapped, never allowed to connect to any network)
+
+You are responsible for keeping your Tails USBs updated. This guide will
+explain that process.
+
+One-Click Updates
+-----------------
+
+For the Internet-connected Tails workstations (the *Admin Workstation* and
+the *Journalist Workstation*), you'll see a notification when a new version
+of Tails is available. We recommend that you :ref:`back up your existing
+configuration<Step 1>` and upgrade as soon as possible.
```

Comment:
Nit: It's not consistent throughout the existing docs, but we tend to use a space between link title and link target.

Reply:
fixed in  #c91e4931f